Common/Common.csproj

```diff
@@ -4,4 +4,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="LiteDB" Version="4.1.4" />
```

Comment:
Can we use sqlite for this?

Reply:
I'd like to, but I had problems with concurrent access to the DB. I'll see if it's still an issue.

Reply:
SQLite adds 4Mb of dlls to the fulltrust process (DBLite is 300Kb), crashes all the time and API is ugly. For reference code for SQLite is in 29ddce0. No SQLite!